src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -326,17 +326,27 @@ public static void TestGetDateTimeFormats()
         Assert.Equal(july28Formats.OrderBy(t => t), actualJuly28Formats.OrderBy(t => t));
     }
 
-
     [Theory]
     [InlineData("fi-FI")]
     [InlineData("nb-NO")]
     [InlineData("nb-SJ")]
+    public static void TestSpecialCulturesParsing(string cultureName)
+    {
+        TestDateTimeParsingWithSpecialCultures(cultureName);
+    }
+
+    [Theory]
     [InlineData("sr-Cyrl-XK")]
     [InlineData("sr-Latn-ME")]
     [InlineData("sr-Latn-RS")]
     [InlineData("sr-Latn-XK")]
-    [ActiveIssue(3391)]
-    public static void TestDateTimeParsingWithSpecialCultures(string cultureName)
+    [PlatformSpecific(PlatformID.Windows)] // ToDo: Serbian cultures in Linux have wrong date patterns. need to fix that.
```

Comment:
Rather than using PlatformSpecific, it seems like we should open an issue to track the fix for Unix, and here use ```[ActiveIssue(issueNumber, PlatformID.AnyUnix]```.